homeassistant/components/frontend/__init__.py

```diff
@@ -135,13 +136,16 @@ def async_register_index_routes(self, router, index_view):
             'get', '/{}/{{extra:.+}}'.format(self.frontend_url_path),
             index_view.get)
 
-    def as_dict(self):
+    def to_response(self, hass, request):
         """Panel as dictionary."""
         return {
             'component_name': self.component_name,
             'icon': self.sidebar_icon,
             'title': self.sidebar_title,
-            'url': self.webcomponent_url,
+            'url':
+                (self.webcomponent_url_latest if
```

Comment:
I have to admit that i don't like these inline if-statements. It makes it difficult to read. I would prefer:

```
info = {
    …
}

if is_latest(…):
    info['url'] = …
else:
    info['url'] = …

return info
```